patches/patch-log4net-csproj-dotnet46.diff

```diff
@@ -28,111 +28,111 @@
--    <PackageReference Include="System.Diagnostics.TraceSource" Version="4.0.0" />
--    <PackageReference Include="System.IO.FileSystem" Version="4.3.0" />
--    <PackageReference Include="System.IO.FileSystem.Watcher" Version="4.0.0" />
--    <PackageReference Include="System.Linq" Version="4.3.0" />
--    <PackageReference Include="System.Net.NameResolution" Version="4.0.0" />
--    <PackageReference Include="System.Net.Requests" Version="4.0.11" />
--    <PackageReference Include="System.Net.Sockets" Version="4.3.0" />
--    <PackageReference Include="System.Reflection" Version="4.3.0" />
--    <PackageReference Include="System.Reflection.Extensions" Version="4.3.0" />
--    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.1.0" />
--    <PackageReference Include="System.Runtime.Extensions" Version="4.3.0" />
--    <PackageReference Include="System.Runtime.InteropServices" Version="4.3.0" />
--    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
--    <PackageReference Include="System.Runtime.Serialization.Formatters" Version="4.3.0" />
--    <PackageReference Include="System.Text.RegularExpressions" Version="4.3.0" />
--    <PackageReference Include="System.Text.Encoding" Version="4.3.0" />
--    <PackageReference Include="System.Threading" Version="4.3.0" />
--    <PackageReference Include="System.Threading.Thread" Version="4.0.0" />
--    <PackageReference Include="System.Threading.Timer" Version="4.3.0" />
--    <PackageReference Include="System.Xml.ReaderWriter" Version="4.3.0" />
--    <PackageReference Include="System.Xml.XmlDocument" Version="4.0.1" />
--  </ItemGroup>
--  <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
--    <PackageReference Include="System.Configuration.ConfigurationManager" Version="4.5.0" />
--  </ItemGroup>
-   <ItemGroup>
-     <BootstrapperPackage Include="Microsoft.Net.Client.3.5">
-       <Visible>False</Visible>
-@@ -190,12 +126,6 @@
-       <Link>package-icon.png</Link>
-     </Content>
-   </ItemGroup>
--  <ItemGroup>
--    <PackageReference Include="Microsoft.Net.Compilers" Version="3.7.0">
--      <PrivateAssets>all</PrivateAssets>
--      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
--    </PackageReference>
--  </ItemGroup>
-   <PropertyGroup />
-   <Import Project="../MonoForFramework.targets" />
- </Project>
+--- src/log4net/log4net.csproj	2022-08-03 07:31:38.000000000 +0100
++++ src/log4net/log4net.csproj	2022-09-27 17:14:16.475061900 +0100
+@@ -30,7 +30,7 @@
+     <RepositoryUrl>https://github.com/apache/logging-log4net</RepositoryUrl>
+     <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+     <Platforms>AnyCPU</Platforms>
+-    <TargetFrameworks>net20;net35;net40;net40-client;net45;netstandard1.3;netstandard2.0</TargetFrameworks>
++    <TargetFrameworks>net46</TargetFrameworks>
```

Comment:
any reason we can't target both 4.8 and 4.6 here, and then build the project once?

similarly to Json.NET, basically

Reply:
Hmmm, not really, let me have a second look.